code/game/objects/items/devices/atomic_disassembler.dm

```diff
@@ -0,0 +1,156 @@
+/obj/item/device/atomic_disassembler
+	name = "P.A.D.D."
```

Comment:
This is an awful name. Ideally it should actually be descriptive about what it does. For example, "atomic disassembly device" is pretty cool and way more grokable about what it is without having to examine it.